.github/workflows/test.yml

```diff
@@ -14,10 +14,11 @@ jobs:
         rails:
           - 5.2.0
           - 6.0.0
-          - 6.1.0
         include:
           - ruby: 2.4
             rails: 5.2.0
+          - ruby: 3.0
+            rails: 6.1.0
```

Comment:
Rails 6.1 should still be tested against Ruby 2.7 because that's our current supported version of Ruby. Do we actually want to include Ruby 3.0 in the ruby matrix instead?

Reply:
I added back `6.1.0` to the testing matrix. Rails introduces support for Ruby 3 only starting from version 6.0.3 https://github.com/rails/rails/issues/40938#issuecomment-751357907